hardware/raspberrypi/booteeprom.md

```diff
@@ -6,104 +6,148 @@ The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains cod
 If an error occurs during boot then an [error code](../../configuration/led_blink_warnings.md) will be displayed via the green LED. Newer versions of the bootloader will display a [diagnostic message](boot_diagnostics.md) which will be shown on both HDMI displays.
 
 ## Updating the bootloader
+### Pi 4 and Pi 400
+Raspberry Pi OS automatically updates the bootloader for critical bug fixes. The recommended methods for manually updating the bootloader or changing the boot modes are the Raspberry Pi Imager and raspi-config.
```

Comment:
I would remove `the` from the front of `Raspberry Pi Imager` and put raspi-config in backticks.

Reply:
Remove 'the' and replaced raspi-config to a link, if only we automated keyword links in Github docs